app/javascript/components/action-button/components/default-button/component.spec.js

```diff
@@ -0,0 +1,56 @@
+import { mountedComponent, screen } from "test-utils";
```

Comment:
Copy and paste these new files into the old files and rename to component.spec.js. This should keep the git history of the file.

Reply:
done